### ISSUE TITLE
setuptools pip wheel failed with error code 1

### ISSUE THREAD

**Issue by tgdn** [certbot/certbot#2887]
I am following [this guide](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-14-04) from DigitalOcean on their servers on Ubuntu 14.04.
When trying to install the certificate with 
`./letsencrypt-auto certonly --webroot --webroot-path /usr/share/nginx/html -d mysite.com -d www.mysite.com`

I get the following:

``` bash
Bootstrapping dependencies for Debian-based OSes...
...
...
dialog is already the newest version.
gcc is already the newest version.
python is already the newest version.
python-dev is already the newest version.
augeas-lenses is already the newest version.
ca-certificates is already the newest version.
libaugeas0 is already the newest version.
libffi-dev is already the newest version.
libssl-dev is already the newest version.
python-virtualenv is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Checking for new version...
Creating virtual environment...
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/virtualenv.py", line 2316, in <module>
    main()
  File "/usr/local/lib/python2.7/dist-packages/virtualenv.py", line 708, in main
    symlink=options.symlink)
  File "/usr/local/lib/python2.7/dist-packages/virtualenv.py", line 941, in create_environment
    download=download,
  File "/usr/local/lib/python2.7/dist-packages/virtualenv.py", line 897, in install_wheel
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=SCRIPT)
  File "/usr/local/lib/python2.7/dist-packages/virtualenv.py", line 792, in call_subprocess
    % (cmd_desc, proc.returncode))
OSError: Command /home/tgdn/.local/sh...ncrypt/bin/python2.7 - setuptools pip wheel failed with error code 1
```

For more information:

``` bash
$ python -V
Python 2.7.6

$ pip -V
pip 8.1.1 from /usr/local/lib/python2.7/dist-packages (python 2.7)
```

This issue is related to this one https://github.com/letsencrypt/letsencrypt/issues/2850.

**Comment by starchy** [comment]
Confirmed this issue is present in Debian Jessie.

**Comment by thomasjonas** [comment]
Same problem here on Ubuntu 16.04... But with python 2.7.11+

**Comment by nyon** [comment]
I had the same problem on Ubuntu 16.04. 

    locale-gen "en_US.UTF-8"
    export LC_ALL="en_US.UTF-8"

fixed it.

**Comment by danger89** [comment]
Same issue with Debian Jessie. I could try the `locale-gen` above. But is this a workaround?

**Comment by bmw** [comment]
@danger89, if you're on Debian Jessie, it is recommended to use the `certbot` packages from `jessie-backports`.

**Comment by danger89** [comment]
@bmw I know however this one was also broken ;). See issue #1080

**Comment by yan12125** [comment]
I've had some bad experiences with Python 2 + virtualenv + exotic locales. Once Python 3 is done (#4507), this issue may disappear as well.

**Comment by bmw** [comment]
I'm closing this as a duplicate of #2883 which has more comments.
